src/haxesharp/exceptions/ArgumentNullException.hx

```diff
@@ -0,0 +1,20 @@
+package haxesharp.exceptions;
+
+/**
+An exception indicating that the given arguments are not valid
+*/
+class ArgumentNullException extends Exception
```

Comment:
Shouldn't this extend `ArgumentException`?

Reply:
You're right; fixed.